source/utils/copyToClipboardOnClick.js

```diff
@@ -0,0 +1,10 @@
+// see https://stackoverflow.com/questions/45071353/copy-text-string-on-click/45071469#45071469
+// copy any string to a clipboard only when a click action is triggered
+export default function copyToClipbaordOnClick(str) {
```

Comment:
Let's rename it to copyToClipboard (there's nothing about onClick here).

Also let's make sure this code is only executed in the browser.